src/appengine/handlers/cron/project_setup.py

```diff
@@ -119,6 +119,12 @@ def job_name(self, project_name):
     minimize_job_override=LIBFUZZER_ASAN_JOB)
 NO_ENGINE_ASAN_JOB = JobInfo('asan_', 'none', 'address', [])
 
+HONGGFUZZ_ASAN_JOB = JobInfo(
+    'honggfuzz_asan_',
+    'honggfuzz',
+    'address', ['honggfuzz', 'engine_asan'],
+    minimize_job_override=LIBFUZZER_ASAN_JOB)
```

Comment:
Eventually we want to support native minimization using fuzzing engine itself, does honggfuzz support that ?

Reply:
I don't think so.. (https://github.com/google/honggfuzz/blob/master/docs/USAGE.md)